cpython-windows/build.py

```diff
@@ -1759,6 +1759,15 @@ def build_cpython(
             log("copying %s to %s" % (source, dest))
             shutil.copyfile(source, dest)
 
+        # Rename to `python.exe` when an alternative executable is built, e.g., when
+        # free-threading is enabled the name is `python3.13t.exe`.
+        canonical_python_exe = install_dir / "python.exe"
```

Comment:
Is `install_dir` the same as `out_dir / "python" / "install"`?

Reply:
I'm just comparing to `python_exe = out_dir / "python" / "install" / python_exe` below.

Reply:
Yep

https://github.com/indygreg/python-build-standalone/blob/00ea84cd2a525c8d20566482633903878bae2996/cpython-windows/build.py#L1655